cmd/kusk/internal/utils/utils.go

```diff
@@ -141,3 +171,71 @@ func IsPodRunning(ctx context.Context, c client.Client, podName, namespace strin
 		return false, nil
 	}
 }
+
+func getCRDClient(ctx context.Context) (*clientset.Clientset, error) {
```

Comment:
This returns a client set from a given config, not specifically a CRD client.

Reply:
this is clientset `"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"` vs `"sigs.k8s.io/controller-runtime/pkg/client"`. But I'm testing to check if this wait is needed at all.